Makefile

```diff
@@ -166,7 +166,7 @@ ci-fmt: ci-lint
 
 ci-tests:
 	@echo "running tests with parametrized platform: $(PARAMETRIZE)"
-	pants test lumigator/python/mzai/backend/tests:backend_tests@parametrize=$(PARAMETRIZE)
+	pants test lumigator/python/mzai/backend/backend/tests:backend_tests@parametrize=$(PARAMETRIZE)
```

Comment:
are there two levels of backend here?

Reply:
yes (confusingly) the inner backend is to make the 'backend.blah` python structure work. The outer `backend` contains the dockerfile, pyproject.toml, etc.